test-data/unit/lib-stub/dataclasses.pyi

```diff
@@ -1,30 +1,86 @@
+import sys
 from typing import Any, Callable, Generic, Mapping, Optional, TypeVar, overload, Type
 
 _T = TypeVar('_T')
 
 class InitVar(Generic[_T]):
     ...
 
+if sys.version_info >= (3, 10):
+    class _KW_ONLY_TYPE: ...
+    KW_ONLY = _KW_ONLY_TYPE
```

Comment:
in the stub from typeshed this line is

```python
    KW_ONLY: _KW_ONLY_TYPE
```

Without making this change the following will fail

```python
[case testDataclassesOrderingKwOnlyWithSentinel]
# flags: --python-version 3.10
from dataclasses import dataclass, KW_ONLY

@dataclass
class Application:
    _: KW_ONLY
    name: str = 'Unnamed'
    rating: int
```

```
error: Variable "dataclasses.KW_ONLY" is not valid as a type
```

Any feedback on this? Does this change need to be made in typeshed or am I specifying the type wrong in the test case?

Reply:
I think typeshed has to change. I believe that test case is how the object is supposed to be used, so typeshed should accommodate it.